internal/provider/provider.go

```diff
@@ -65,7 +67,23 @@ func New(version string) func() *schema.Provider {
 					Optional:    true,
 					Default:     "",
 				},
-
+				"assume_role": {
+					Description: "Settings for making use of the AWS Assume Role Function",
+					Type:        schema.TypeList,
+					Required:    false,
+					Optional:    true,
+					MaxItems:    1,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"role_arn": {
+								Description: "The ARN to be used with Assume Role. If available.",
+								Type:        schema.TypeString,
+								Optional:    true,
+								Default:     "",
```

Comment:
I believe you don't need a default when the property is optional

Reply:
Thanks